orgs/trial.json

```diff
@@ -1,6 +1,6 @@
 {
   "orgName": "NPSP - Trial Org",
-  "template": "0TT1I000006uGCr",
```

Comment:
Something is still not right here. The `trial.json` shouldn't be in the PR at all as changed in any way.

Reply:
Sorry. You can ignore these last two comments.